CONTRIBUTING.md

```diff
@@ -0,0 +1,22 @@
+# Contributing
+
+To make contributions to this charm, you'll need a working [Juju development setup](https://juju.is/docs/sdk/dev-setup).
```

Comment:
Do we expect to have a charm in this project?

Reply:
Fixed, no